prepare/templates/qa/contextual.py

```diff
@@ -1,15 +1,19 @@
 from src.unitxt.catalog import add_to_catalog
-from src.unitxt.templates import InputOutputTemplate, TemplatesList
+from src.unitxt.templates import MultiReferenceTemplate, TemplatesList
 
 add_to_catalog(
-    InputOutputTemplate(input_format="Context: {context}\nQuestion: {question}", output_format="{answer}"),
+    MultiReferenceTemplate(
+        input_format="Context: {context}\nQuestion: {question}", output_format="{answers}", references_field="answers"
```

Comment:
We should not require output format in MultiReferenceTemplate.  This is because it is not used.

Reply:
done